constants/src/lib.rs

```diff
@@ -236,6 +236,8 @@ pub const MONGO_CAST_SUPPORT: u32 = SQL_CVT_CHAR
     | SQL_CVT_REAL
     | SQL_CVT_DOUBLE
     | SQL_CVT_VARCHAR
+    | SQL_CVT_WLONGVARCHAR
```

Comment:
We should add LONGVARCHAR and WCHAR to this PR, for completeness, or shall I say to cover everything we might get from PowerBI.

Reply:
uhm, I think it's not just here that the change is needed, but everywhere else in this PR.
It needs to be added in bson_type_info.rs, type_info.rs and in the tests.
Otherwise we will end up with the same problem where PowerBI want to cast a value to LONGVARCHAR or WCHAR and it fails because there is no LONGVARCHAR and WCHAR types in SQLGetTypeInfo, no?
I suppose that we should map them to the "string" name too 🤔
Maybe we should try to do more tests, because we list SQL_CVT_CHAR, but there is no SQL_CHAR reported by SQLGetTypeInfo. I have no way to tell if PowerBI will map something to SQL_CHAR or not. It is technically the data type for fixed length string, so if data is consistent in length, they might be using it, like 'Y', 'N' for boolean representation or something like that.

Reply:
WCHAR is there, it's missing LONGVARCHAR however